src/map.cpp

```diff
@@ -174,7 +175,7 @@ bool Map::isNodeUnderground(v3s16 p)
 bool Map::isValidPosition(v3s16 p)
 {
 	v3s16 blockpos = getNodeBlockPos(p);
-	MapBlock *block = getBlockNoCreate(blockpos);
+	MapBlock *block = getBlockNoCreateNoEx(blockpos);
```

Comment:
Huh, this isn't related to the PR... should it be a separate commit instead, despite its size?